eng/pipelines/templates/steps/build.yml

```
@@ -54,7 +54,7 @@ steps:
           echo "##vso[task.setvariable variable=rushBuildCacheCred;issecret=true;]$sasToken"
 
   - pwsh: |
-      node eng/tools/rush-runner/index.js build $(ChangedServices) -packages "$(ArtifactPackageNames)" --verbose -p max
+      node eng/tools/rush-runner/index.js build "$(ChangedServices)"" -packages "$(ArtifactPackageNames)" --verbose -p max
```

we only need one quote, right?
```suggestion
      node eng/tools/rush-runner/index.js build "$(ChangedServices)" -packages "$(ArtifactPackageNames)" --verbose -p max
```